ocamldoc/stdlib_non_prefixed/extract_pervasives.awk

```diff
@@ -0,0 +1,9 @@
+# This script extract the Pervasives submodule from stdlib.mli into
```

Comment:
This file either needs a copyright header, or `ocaml-typo=missing-header` in .gitattributes (I think `missing-header` is fine)